docs/api/create-list.md

```diff
@@ -76,7 +76,7 @@ keystone.createList('User',
 
 ### `mutations`
 
-An Array of custom mutations in the form of: `{ schema, resolver }.
+An Array of custom mutations in the form of: \`{ schema, resolver }.
```

Comment:
Dafuq?

Reply:
```suggestion
An Array of custom mutations in the form of: `{ schema, resolver }`.
```

Reply:
Oh, that'll be `yarn format` trying to deal with a dangling backtick. Let me sort that out.